sdk/metric/pipeline_test.go

```diff
@@ -518,3 +536,91 @@ func TestExemplars(t *testing.T) {
 		check(t, r, 2, 2, 2)
 	})
 }
+
+func TestAddingAndObservingMeasureConcurrentSafe(t *testing.T) {
+	exp := &fnExporter{}
+	r1 := NewPeriodicReader(exp, WithInterval(10*time.Millisecond))
+	r2 := NewPeriodicReader(exp, WithInterval(10*time.Millisecond))
+
```

Comment:
This tests leaks two periodic readers goroutine.
Do we need to use a period reader for this test? Cannot it be a manual reader?

Reply:
Switched to ManualReader